### PR TITLE
Review log formatting to use parameterized messages.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

**Review by ylangisc — COMMENTED** [review]
`isDebugEnabled()` checks can be omitted now as no evaluation happens this way.

File: core/src/main/java/ch/cyberduck/core/threading/DefaultFailureDiagnostics.java [hunk 1]
```diff
@@ -49,9 +49,7 @@ public final class DefaultFailureDiagnostics implements FailureDiagnostics<Backg
 
     @Override
     public Type determine(final BackgroundException failure) {
-        if(log.isDebugEnabled()) {
-            log.debug(String.format("Determine cause for failure %s", failure));
-        }
+        log.debug("Determine cause for failure {}", failure.getMessage());
```
Comment (ylangisc) [inline core/src/main/java/ch/cyberduck/core/threading/DefaultFailureDiagnostics.java:52]: NPE for `null` failure.

**Review by ylangisc — CHANGES_REQUESTED** [review]

File: core/src/main/java/ch/cyberduck/core/threading/DefaultFailureDiagnostics.java (hunk 1 above)
Comment (dkocher) [reply core/src/main/java/ch/cyberduck/core/threading/DefaultFailureDiagnostics.java:52]: Argument is  not expected to be `null`.

Comment (ylangisc) [reply core/src/main/java/ch/cyberduck/core/threading/DefaultFailureDiagnostics.java:52]: https://github.com/iterate-ch/cyberduck/blob/1034b879dd69a231c22851d2eeac438c9ec7a856/core/src/test/java/ch/cyberduck/core/threading/DefaultFailureDiagnosticsTest.java#L34

**Review by ylangisc — APPROVED** [review]

**Pull request merged**